.sops.yaml

```diff
@@ -9,6 +9,7 @@ keys:
   - &ryantm age1d87z3zqlv6ullnzyng8l722xzxwqr677csacf3zf3l28dau7avfs6pc7ay
   - &zimbatm age1jrh8yyq3swjru09s75s4mspu0mphh7h6z54z946raa9wx3pcdegq0x8t4h
   - &zowoq age1m7xhem3qll35d539f364pm6txexvnp6k0tk34d8jxu4ry3pptv7smm0k5n
+  - &adisbladis age1yubikey1qt7xflk6z7sqp28tuyp0y239l9dklllklaxffqqln8dtvvurxp4m5hgx9nd
```

Comment:
Seems `sops` doesn't support this type of key.

```
failed to parse input as Bech32-encoded age public key: malformed recipient "age1yubikey1qt7xflk6z7sqp28tuyp0y239l9dklllklaxffqqln8dtvvurxp4m5hgx9nd": invalid type "age1yubikey"
```

I'd prefer to not re-add the gpg stuff, TLDR: for me it is just extra stuff to download every time I use the shell.nix.

That said if you can't use a regular age key you can see sops gpg hook that you'll need to add in https://github.com/nix-community/infra/commit/92a43cb33d88ad5983f34f26818a0f23e0ebf03c.

Reply:
Could you use your current ssh key instead of generating another key? https://github.com/Mic92/ssh-to-age

Reply:
Yes. Sops does not support yubikey keys

Reply:
> Could you use your current ssh key instead of generating another key? https://github.com/Mic92/ssh-to-age

I do not use file based ssh keys. I use `gpg-agent` and my key resides in hardware.

Reply:
So the status quo is that because GPG keys are unsupported here for some reason I fail to understand, the yubikey age plugin isn't supported by sops hardware based keys are not supported.

Reply:
Would it be ok to generate an age key specifically to deploy to the nix-community? It doesn't strictly need to be correlated with your SSH key AFAIK.

Most of us aren't using GPG so we saw an opportunity to remove it from our system config. I think you were AWOL at the time we were discussing this so we missed your input.

Reply:
I would be OK adding gnupg support back.

Reply:
Same, I still have GPG installed due to gopass

Reply:
Opposed to re-adding it just for 1/5 users and 1/11 total keys.

Reply:
> Would it be ok to generate an age key specifically to deploy to the nix-community? It doesn't strictly need to be correlated with your SSH key AFAIK.

That would be OK as a temporary workaround while we figure out what to do with this.

> Opposed to re-adding it just for 1/5 users and 1/11 total keys.

I understand you see it as a burden, but right now it looks like the only remotely secure way to approach key management with sops.

Reply:
I've added a software based key for now :/

Reply:
My age key doesn't even have a passphrase because it's not supported very well by sops. https://github.com/getsops/sops/issues/933 . And GPG is quite archaic and has tons of UX problems.

It would be nice to find a solution that supports hardware dongles / 2FA

Reply:
@adisbladis if you use a commandline password manager you can also do:

```
export SOPS_AGE_KEY=$(pass some-key)
```